python/cli/prelude_cli/views/interactive.py

```diff
@@ -873,7 +874,9 @@ def __init__(self, wiz: Wizard, title: str):
         self.title = title
 
     def enter(self):
-        webbrowser.open('https://platform.preludesecurity.com/detect', new=2)
+        account = self.wiz.account.headers['account']
+        token = self.wiz.account.headers['token']
+        webbrowser.open(f'https://platform.preludesecurity.com?account={account}&token={token}', new=2)
```

Comment:
@kenrick thoughts?

Reply:
Yeah I asked @mireaulf to use `#` instead of a query string `?` this way the browser will not keep the account and token in its history or send it the server